pkg/config/config.go

```diff
@@ -343,11 +343,19 @@ func LoadEnvFileWithPath(path string, override bool) {
 }
 
 // LoadEnvFile .
-func LoadEnvFile() {
+func LoadEnvFile(profile string) {
 	wd, err := os.Getwd()
 	if err != nil {
 		return
 	}
 	path := filepath.Join(wd, ".env")
 	LoadEnvFileWithPath(path, false)
+	LoadEnvFileByProfile(path, profile)
```

Comment:
no need new `LoadEnvFileByProfile`, just adjust env file path by profile.
I prefer to use format separated by `.`, such as: `.env.dev`, `.env.daily`.

Reply:
I think new LoadEnvFileByProfile is better.